mine.py

```diff
@@ -1,7 +1,10 @@
 # Write a line of code that prints "Hello world!".
+print('"Hello world!"')
 
 
 def hello_world():
   # Create a variable that has the value of "Hello world!" so that the function will return the appropriate value.
+  variable_name = '"Hello world!"'
```

Comment:
```suggestion
  variable_name = 'Hello world!'
```
Need to remove the extra double quotes